third_party/istio-1.1.1/download-istio.sh

```diff
@@ -68,6 +69,7 @@ helm template --namespace=istio-system \
   `# Set gateway pods to 1 to sidestep eventual consistency / readiness problems.` \
   --set gateways.istio-ingressgateway.autoscaleMin=1 \
   --set gateways.istio-ingressgateway.autoscaleMax=1 \
+  --set gateways.istio-ingressgateway.sds.enabled=true \
```

Comment:
Is this a setting that we should note alongside the knob to enable auto-TLS?

Reply:
Yes, we should note this setting. I will note this setting in my PR https://github.com/knative/serving/pull/3585.